### ISSUE TITLE
OpenJDK 8 versions?

### ISSUE THREAD

**Issue by daincredibleholg** [halcyon/asdf-java#39]
Unfortunately, from time to time, Java 8 is still needed and I was wondering if would be possible to provide OpenJDK 8 builds?

**Comment by halcyon** [comment]
I too rely upon Java 8. Oracle has been erecting barriers to automating downloads from their site for some time. This is why I rewrote asdf-java around https://adoptopenjdk.net/. 

May I suggest:

```
 % asdf install java adopt-openjdk-8u232-b09                                                                                               
############################################################################################################################################ 100.0%
OpenJDK8U-jdk_x64_linux_hotspot_8u232b09.tar.gz
############################################################################################################################################ 100.0%
OpenJDK8U-jdk_x64_linux_hotspot_8u232b09.tar.gz.sha256.txt
OpenJDK8U-jdk_x64_linux_hotspot_8u232b09.tar.gz: OK
```

**Comment by daincredibleholg** [comment]
Interestingly, this doesn't show up in `list-all`:
```
➤  asdf list-all java
oracle-8.131
oracle-8.141
oracle-8.191
oracle-11.0.1
openjdk-10.0.2
openjdk-11
openjdk-11.0.1
```

**Comment by halcyon** [comment]
Sounds like you might be running an outdated version of `asdf-java`, as these candidates are not offered in the current version of `asdf-java`.

```
% asdf list-all java
adopt-openjdk-10.0.2+13
adopt-openjdk-10.0.2+13.1
adopt-openjdk-11+28
adopt-openjdk-11.0.1+13
adopt-openjdk-11.0.2+7
adopt-openjdk-11.0.2+9
adopt-openjdk-11.0.3+7
adopt-openjdk-11.0.4+11
adopt-openjdk-11.0.4+11.2
adopt-openjdk-11.0.4+11.4
adopt-openjdk-11.0.5+10
adopt-openjdk-12+33
adopt-openjdk-12.0.1+12
adopt-openjdk-12.0.2+10
adopt-openjdk-12.0.2+10.2
adopt-openjdk-12.0.2+10.3
adopt-openjdk-13+33
adopt-openjdk-13.0.1+9
adopt-openjdk-8u172-b11
adopt-openjdk-8u181-b13
adopt-openjdk-8u191-b12
adopt-openjdk-8u192-b12
adopt-openjdk-8u202-b08
adopt-openjdk-8u212-b03
adopt-openjdk-8u212-b04
adopt-openjdk-8u222-b10
adopt-openjdk-8u232-b09
adopt-openjdk-9+181
adopt-openjdk-9.0.4+11
amazon-corretto-11.0.3.7.1
amazon-corretto-11.0.4.11.1
amazon-corretto-11.0.5.10.1
amazon-corretto-11.0.5.10.2
amazon-corretto-8.212.04.2
amazon-corretto-8.232.09.1
amazon-corretto-8.232.09.2
azul-zulu-11.31.11-jdk11.0.3
azul-zulu-11.35.13-jdk11.0.5
azul-zulu-11.35.15-jdk11.0.5
azul-zulu-12.2.3-jdk12.0.1
azul-zulu-13.27.9-jdk13
azul-zulu-13.28.11-jdk13.0.1
azul-zulu-7.29.0.5-jdk7.0.222
azul-zulu-7.34.0.5-jdk7.0.242
azul-zulu-8.42.0.21-jdk8.0.232
azul-zulu-8.42.0.23-jdk8.0.232
azul-zulufx-11.35.15-jdk11.0.5
azul-zulufx-8.42.0.23-jdk8.0.232
adopt-openjdk-10.0.2+13_openj9-0.9.0
adopt-openjdk-11.0.2+9_openj9-0.12.1
adopt-openjdk-11.0.3+7_openj9-0.14.0
adopt-openjdk-11.0.3+7_openj9-0.14.3
adopt-openjdk-11.0.4+11.2_openj9-0.15.1
adopt-openjdk-11.0.4+11.4_openj9-0.15.1
adopt-openjdk-11.0.4+11_openj9-0.15.1
adopt-openjdk-11.0.5+10.1_openj9-0.17.0
adopt-openjdk-11.0.5+10_openj9-0.17.0
adopt-openjdk-12+33_openj9-0.13.0
adopt-openjdk-12.0.1+12_openj9-0.14.1
adopt-openjdk-12.0.2+10.2_openj9-0.15.1
adopt-openjdk-12.0.2+10.3_openj9-0.15.1
adopt-openjdk-12.0.2+10_openj9-0.15.1
adopt-openjdk-13+33_openj9-0.16.0
adopt-openjdk-13.0.1+9.1_openj9-0.17.0
adopt-openjdk-13.0.1+9_openj9-0.17.0
adopt-openjdk-8u162-b12_openj9-0.8.0
adopt-openjdk-8u181-b13_openj9-0.9.0
adopt-openjdk-8u192-b12_openj9-0.11.0
adopt-openjdk-8u202-b08_openj9-0.12.1
adopt-openjdk-8u212-b03_openj9-0.14.0
adopt-openjdk-8u212-b04_openj9-0.14.2
adopt-openjdk-8u222-b10_openj9-0.15.1
adopt-openjdk-8u232-b09.1_openj9-0.17.0
adopt-openjdk-8u232-b09_openj9-0.17.0
adopt-openjdk-9.0.4+12_openj9-0.9.0
adopt-openjdk-11.0.2+9_openj9-0.12.1_large-heap
adopt-openjdk-11.0.3+7_openj9-0.14.0_large-heap
adopt-openjdk-11.0.3+7_openj9-0.14.3_large-heap
adopt-openjdk-11.0.4+11.2_openj9-0.15.1_large-heap
adopt-openjdk-11.0.4+11.4_openj9-0.15.1_large-heap
adopt-openjdk-11.0.4+11_openj9-0.15.1_large-heap
adopt-openjdk-11.0.5+10.1_openj9-0.17.0_large-heap
adopt-openjdk-11.0.5+10_openj9-0.17.0_large-heap
adopt-openjdk-12+33_openj9-0.13.0_large-heap
adopt-openjdk-12.0.1+12_openj9-0.14.1_large-heap
adopt-openjdk-12.0.2+10.2_openj9-0.15.1_large-heap
adopt-openjdk-12.0.2+10_openj9-0.15.1_large-heap
adopt-openjdk-13+33_openj9-0.16.0_large-heap
adopt-openjdk-13.0.1+9.1_openj9-0.17.0_large-heap
adopt-openjdk-13.0.1+9_openj9-0.17.0_large-heap
adopt-openjdk-8u181-b13_openj9-0.9.0_large-heap
adopt-openjdk-8u192-b12_openj9-0.11.0_large-heap
adopt-openjdk-8u202-b08_openj9-0.12.1_large-heap
adopt-openjdk-8u212-b03_openj9-0.14.0_large-heap
adopt-openjdk-8u212-b04_openj9-0.14.2_large-heap
adopt-openjdk-8u222-b10_openj9-0.15.1_large-heap
adopt-openjdk-8u232-b09.1_openj9-0.17.0_large-heap
adopt-openjdk-8u232-b09_openj9-0.17.0_large-heap
```

**Comment by daincredibleholg** [comment]
According to `asdf` I am all up to date:

```
➤  asdf plugin update java
Updating java...
Already on 'master'
Your branch is up to date with 'origin/master'.
```

```
➤  asdf update
HEAD is now at 6207e42 Update version to 0.7.6
Updated asdf to release v0.7.6
```

**Comment by halcyon** [comment]
And if you were to run the following do you get this?
```
% cd ~/.asdf/plugins/java && git remote get-url origin                                                                                    
git@github.com:halcyon/asdf-java
```

**Comment by daincredibleholg** [comment]
It used https://github.com/skotchpine/asdf-java.git as remote URL. I replaced it with

```
cd ~/.asdf/plugins/java && git remote set-url origin git@github.com:halcyon/asdf-java
```

Now it does show me much more versions.

Apologies for not reading the README thoroughly!

**Comment by halcyon** [comment]
No worries, completely understandable considering the change in maintainers of the plugin. Glad it's showing you more choices, looking forward to hearing if the rewrite resolves #40
